benches/component-sharing.rs

```diff
@@ -19,11 +19,11 @@ fn cell_share_bench(c: &mut Criterion) {
                         let name =
                             format!("benches/component-sharing/{}.futil", name);
                         let bench = Path::new(&name);
-                        let lib = Path::new(".");
+                        let lib = [Path::new(".").to_path_buf()];
```

Comment:
Silly nit that you can feel free to ignore. But might be better to just use `PathBuf::from` instead